src/helpers/map.js

```diff
@@ -59,7 +59,7 @@ export function isTraversable(tileID) {
     case tileTypes.BLUE_GATE:
       return amBlue();
     default:
-      return false;
+      throw new Error(`Unknown tileID given function: ${tileID}`);
```

Comment:
oh i think i meant to say "given to function" or something. yeah i'll take the "function" word out